src/Components/Navbar.jsx

```diff
@@ -20,60 +28,15 @@ const Navbar = ({ setCategory }) => {
           </button>
           <div className="collapse navbar-collapse" id="navbarNav">
             <ul className="navbar-nav">
-              <li className="nav-item">
-                <div
-                  className="nav-link"
-                  onClick={() => {
-                    setCategory("technology");
-                  }}>
-                  Technology
-                </div>
-              </li>
-              <li className="nav-item">
-                <div
-                  className="nav-link"
-                  onClick={() => {
-                    setCategory("business");
-                  }}>
-                  Business
-                </div>
-              </li>
-              <li className="nav-item">
-                <div
-                  className="nav-link"
-                  onClick={() => {
-                    setCategory("health");
-                  }}>
-                  Health
-                </div>
-              </li>
-              <li className="nav-item">
-                <div
-                  className="nav-link"
-                  onClick={() => {
-                    setCategory("science");
-                  }}>
-                  Science
-                </div>
-              </li>
-              <li className="nav-item">
-                <div
-                  className="nav-link"
-                  onClick={() => {
-                    setCategory("sports");
-                  }}>
-                  Sports
-                </div>
-              </li>
-              <li className="nav-item">
+              {navbar.links.map((links, index)=>(
+                 <li key={index} className="nav-item btn">
```

Comment:
Instead of use a DRY code make a common file like utils and export our links

Reply:
Thankyou Zamam. Nice work